server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -131,6 +134,14 @@ protected void configure() {
         Multibinder<Version> supportedVersions = Multibinder.newSetBinder(binder(), Version.class);
         supportedVersions.addBinding().toInstance(Version.DRAFT);
         supportedVersions.addBinding().toInstance(Version.RFC8621);
+
+        Multibinder<Capability> supportedCapabilities = Multibinder.newSetBinder(binder(), Capability.class);
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.coreCapability(JmapRfc8621Configuration.UPLOAD_LIMIT_30_MB()));
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.MAIL_CAPABILITY());
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.QUOTA_CAPABILITY());
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.SHARES_CAPABILITY());
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.VACATION_RESPONSE_CAPABILITY());
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.SUBMISSION_CAPABILITY());
```

Comment:
Just a remark: Should we eventually one day move pieces of code like that one that combines draft and rfc out of the draft package, as it's supposed to get deprecated in future?